CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 - `[jest-environment-jsdom]` [**BREAKING**] Upgrade jsdom to 19.0.0 ([#12290](https://github.com/facebook/jest/pull/12290))
 - `[jest-environment-jsdom]` [**BREAKING**] Add default `browser` condition to `exportConditions` for `jsdom` environment ([#11924](https://github.com/facebook/jest/pull/11924))
 - `[jest-environment-node]` [**BREAKING**] Add default `node` and `node-addon` conditions to `exportConditions` for `node` environment ([#11924](https://github.com/facebook/jest/pull/11924))
-- `[@jest/expect]` [**BREAKING**] New module which extends `expect` with `jest-snapshot` matchers ([#12404](https://github.com/facebook/jest/pull/12404))
```

Comment:
new module in itself isn't breaking 🙂

Reply:
Hm.. In a way `expect` received minimal breaking change. That was the reason.

Reply:
yeah, I added a separate entry for it below. Any other breaking changes than the type removal?

Reply:
The breakage I had in mind is this funny line (;

https://github.com/facebook/jest/blob/199f9811ae68b15879cbe18b7ef7ebd61eefcf23/packages/expect/src/index.ts#L434

Reply:
right. that probably should have `thrown` all along, not be a no-op. Can call it out, tho